core/classifiers/CodeClassifier/CodeClassifier.py

```diff
@@ -709,7 +716,12 @@ def train(self, training_data):
 
     # pylint: enable=too-many-locals
     # pylint: disable=too-many-branches, no-self-use
-    def validate(self, classifier_data):
+    # CodeClassifier has been deprecated as it does not support protobuf based
```

Comment:
If it's deprecated, should we just delete the whole thing rather than keep the code around just to support the test cases?

Reply:
Done